web_ui_src/main.js

```diff
@@ -30,6 +30,6 @@ window.SHC_WIDGET_TYPES = WIDGET_TYPES;
 
 // Set up UI with Semantic UI components
 $(function() {
-    $('.main-menu .ui.dropdown').dropdown();
+    $('.main-menu .ui.dropdown').dropdown({on: 'hover'});
```

Comment:
Personal preference, quicker for navigation.  Let me know if you rather want to have it removed.

Reply:
Does it work reasonably well with touch (or hybrid) devices?

Reply:
Feature removed, see above.